### ISSUE TITLE
`operator-no-unspaced` registering * as operator although it is used as a selector

### ISSUE THREAD

**Issue by alexmccabe** [stylelint-scss/stylelint-scss#989]
In #709 this bug was resolved for another selector type, however when used within the `@scope` at-rule it throws the same error. Obviously workaround is to disable next line, but it's a false-positive.

CSS:
```ccss
@scope([class*="my-selector-"]) {
  :scope {
    // Override the default theme variables
    --d-font-size-base: 14;
  }
}
```

Error:
![CleanShot 2024-04-29 at 17 59 11](https://github.com/stylelint-scss/stylelint-scss/assets/2529110/264572da-89b6-46f9-9808-6dc483fd1a1c)

**Comment by kristerkari** [comment]
Thanks for reporting!
